scipy/signal/lti_conversion.py

```diff
@@ -459,6 +465,25 @@ def cont2discrete(system, dt, method="zoh", alpha=None):
         cd = c
         dd = d
 
+    elif method == 'foh':
+        # Build an exponential matrix similar to 'zoh' method
```

Comment:
if you get the shapes beforehand with `m, n = b.shape[1], a.shape[0]` at the top, you don't need to repeat the shape calls